packages/medusa/src/strategies/cart-completion.ts

```diff
@@ -4,14 +4,17 @@ import { EntityManager } from "typeorm"
 import { IdempotencyKey, Order } from "../models"
```

Comment:
praise: Really like the way you've refactored the strategy into isolated methods